pipelines/assets/livelihood_activity.py

```diff
@@ -1337,6 +1337,9 @@ def get_annotated_instances_from_dataframe(
         # Annotate the output metadata with completeness information
         # Get the summary dataframe, grouped by strategy_type
         summary_df = pd.DataFrame(reported_summary_output.value["LivelihoodActivity"])
+        for col in ["income", "expenditure", "kcals_consumed"]:
+            summary_df[col] = pd.to_numeric(summary_df[col], errors="coerce")
+
```

Comment:
@girum-air Do we need to add `.fillna(0)` so that `groupby().sum()` below doesn't return `nan`?